frontend/js/src/user/Listens.tsx

```diff
@@ -744,6 +752,17 @@ export default class Listens extends React.Component<
         />
       );
     }
+    if (canManuallyMap) {
+      additionalMenuItems.push(
+        <ListenControl
+          text="Map to a MusicBrainz Recording"
+          icon={faPencilAlt}
```

Comment:
Same icon as "Write a review", although we only show this if there is an MBID, so perhaps only one would show

Reply:
changed to "Link to MusicBrainz" with a chain icon